src/locker/tests/idle-counter.test.js

```diff
@@ -0,0 +1,120 @@
+import createIdleCounter from '../idle-counter';
+
+jest.useFakeTimers();
+
+const mockDate = (nowIncrement = 1000) => {
```

Comment:
Suggestion: we could use https://github.com/paed01/chronokinesis or similar. I don't if it's worth it at this point, perhaps if we are going to mock dates in future tests?

Reply:
Yes, right now it might be a little too much, but we can try to use it so that in the future it is already available. No strong feelings about it. What do you think?

Reply:
Let’s discuss this in person tomorrow!

Reply:
Let's keep it simple for now then.